docs/glossary.md

```diff
@@ -60,156 +61,156 @@ associated with it called the GFID. This is analogous to inode in a
 
 **Geo-Replication**
-:  Geo-replication provides a continuous, asynchronous, and incremental
-   replication service from site to another over Local Area Networks
-   (LANs), Wide Area Network (WANs), and across the Internet.
-
+: Geo-replication provides a continuous, asynchronous, and incremental
+replication service from site to another over Local Area Networks
+(LANs), Wide Area Network (WANs), and across the Internet.
 
 **Infiniband**
-    InfiniBand is a switched fabric computer network communications link
-    used in high-performance computing and enterprise data centers.
+InfiniBand is a switched fabric computer network communications link
+used in high-performance computing and enterprise data centers.
 
 **Metadata**
-:  Metadata is defined as data providing information about one or more
-   other pieces of data. There is no special metadata storage concept in
-   GlusterFS. The metadata is stored with the file data itself usually in the
-   form of extended attributes
+: Metadata is defined as data providing information about one or more
+other pieces of data. There is no special metadata storage concept in
+GlusterFS. The metadata is stored with the file data itself usually in the
+form of extended attributes
 
 **Namespace**
-:  A namespace is an abstract container or environment created to hold a
-   logical grouping of unique identifiers or symbols. Each Gluster volume
-   exposes a single namespace as a POSIX mount point that contains every
-   file in the cluster.
+: A namespace is an abstract container or environment created to hold a
+logical grouping of unique identifiers or symbols. Each Gluster volume
+exposes a single namespace as a POSIX mount point that contains every
+file in the cluster.
 
 **Node**
-:   A server or computer that hosts one or more bricks.
+: A server or computer that hosts one or more bricks.
 
 **N-way Replication**
-:   Local synchronous data replication which is typically deployed across campus
-    or Amazon Web Services Availability Zones.
+: Local synchronous data replication which is typically deployed across campus
+or Amazon Web Services Availability Zones.
 
 **Petabyte**
-:   A petabyte (derived from the SI prefix peta- ) is a unit of
-    information equal to one quadrillion (short scale) bytes, or 1000
-    terabytes. The unit symbol for the petabyte is PB. The prefix peta-
-    (P) indicates a power of 1000:
+: A petabyte (derived from the SI prefix peta- ) is a unit of
+information equal to one quadrillion (short scale) bytes, or 1000
+terabytes. The unit symbol for the petabyte is PB. The prefix peta-
+(P) indicates a power of 1000:
 
-    1 PB = 1,000,000,000,000,000 B = 10005 B = 1015 B.
+```{ .text .no-copy }
+1 PB = 1,000,000,000,000,000 B = 10005 B = 1015 B.
 
-    The term "pebibyte" (PiB), using a binary prefix, is used for the
-    corresponding power of 1024.
+The term "pebibyte" (PiB), using a binary prefix, is used for the
+corresponding power of 1024.
+```
```

Comment:
Uptill here the text is taken as a block

Reply:
This was the case earlier also, please refer: https://docs.gluster.org/en/latest/glossary/ (current version without this patch)